x-pack/plugins/transform/public/app/sections/transform_management/components/transform_list/action_delete.tsx

```diff
@@ -71,17 +85,96 @@ export const DeleteAction: FC<DeleteActionProps> = ({ items, forceDisable }) =>
     defaultMessage: 'Delete {transformId}',
     values: { transformId: items[0] && items[0].config.id },
   });
-  const bulkDeleteModalMessage = i18n.translate(
-    'xpack.transform.transformList.bulkDeleteModalBody',
-    {
-      defaultMessage:
-        "Are you sure you want to delete {count, plural, one {this} other {these}} {count} {count, plural, one {transform} other {transforms}}? The transform's destination index and optional Kibana index pattern will not be deleted.",
-      values: { count: items.length },
-    }
+  const bulkDeleteModalMessage = (
```

Comment:
Now that this isn't just a single message anymore, suggest to rename it to something without `Message`. I could even become its own component.